main/src/ca/uwaterloo/flix/language/phase/jvm/GenExpression.scala

```diff
@@ -1801,6 +1829,59 @@ object GenExpression {
 
   }
 
+  private def compileInt64(visitor: MethodVisitor, l: Long): Unit = {
+    l match {
+      case -1 =>
+        visitor.visitInsn(ICONST_M1)
+        visitor.visitInsn(I2L) // Sign extend to Long
+
+      case 0 =>
+        visitor.visitInsn(LCONST_0)
+
+      case 1 =>
+        visitor.visitInsn(LCONST_1)
+
+      case 2 =>
+        visitor.visitInsn(ICONST_2)
+        visitor.visitInsn(I2L) // Sign extend to Long
```

Comment:
Why is this not needed for const0 and 1? Seems inconsistent.

Reply:
Because they are longs and these are ints. Apparently, only 0 and 1 are available as long constants

Reply:
😮